iTunesUDataPurge/README.txt

```diff
@@ -1,16 +1,29 @@
 This folder is to store useful standalone programs to access/modify iTunesU tool data.
 
+To compile and build executables, do:
+
+mvn package
+
+It will generate jar file iTunesuDataPurge-<VERSION>-jar-with-dependencies.jar in the target folder.
+
 Java commands to download iTunesU tool content or remove iTunesU tools:
 
-1. DownloadITunesuFile.java downloads itunesu content to local drive, based on given list of site ids. Before use, one need to change the program for required credentials and server information. 
+1. ReadITunesU.java parses the iTunes U showTree file, based on a list of sakai site ids. 
+
+It accepts one parameter, site_id.txt, with line fomat of <site_name>/t<site_id>
+
+The result is a tab-spearated text file, showing the total file storage size used by each site page on iTunes U.
+
+java -cp target/iTunesuDataPurge-1.0-jar-with-dependencies.jar ReadITunesU  site_id.txt
+
 
-java -classpath . DownloadITunesuFile site_id_list_file download_folder_path
+2. DownloadITunesuFile.java downloads itunesu content to local drive, based on given list of site ids. 
 
+Before use, one need to change the program for required credentials and server information. 
 
-2. RemoveITunesUCourses.java remove the itunesu course from iTunesu server, based on a given list of site ids. Before use, one need to change the program for required credentials and server information. 
+java -cp target/iTunesuDataPurge-1.0-jar-with-dependencies.jar DownloadITunesuFile site_id.txt <download_working_path>
 
- java -classpath . RemoveITunesUCourses site_id_list_file
 
-3. ReadITunesU.java parses the iTunes U showTree file, based on a list of sakai site ids. The result is a tab-spearated text file, showing the total file storage size used by each site page on iTunes U.
+3. RemoveITunesUCourses.java remove the itunesu course from iTunesu server, based on a given list of site ids. Before use, one need to change the program for required credentials and server information. 
 
-java -classpath . ReadITunesU showtree_file site_id_list_file > itunesu_storage_file
+<TODO>
```

Comment:
This is a placeholder for TLPSO-211 "Update RemoveITunesUCourses to remove both data and page/tools within CTools". Have not decide on the parameter list yet.